File: CS/Dashboard_UnderlyingDataWin/SalesDashboard.cs
using System;
using Dashboard_UnderlyingDataWin.nwindDataSetTableAdapters;

namespace Dashboard_UnderlyingDataWin {
    public partial class SalesDashboard : DevExpress.DashboardCommon.Dashboard {
        public SalesDashboard() {
            InitializeComponent();
        }

        private void SalesDashboard_DashboardLoading(object sender, EventArgs e) {
            SalesPersonTableAdapter categories = new SalesPersonTableAdapter();
            categories.Fill(this.nwindDataSet1.SalesPerson);
        }
    }
}

File: VB/Dashboard_UnderlyingDataWin/SalesDashboard.vb
Imports System
Imports Dashboard_UnderlyingDataWin.nwindDataSetTableAdapters

Namespace Dashboard_UnderlyingDataWin
    Partial Public Class SalesDashboard
        Inherits DevExpress.DashboardCommon.Dashboard

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub SalesDashboard_DashboardLoading(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.DashboardLoading
            Dim categories As New SalesPersonTableAdapter()
            categories.Fill(Me.nwindDataSet1.SalesPerson)
        End Sub
    End Class
End Namespace
